### DISCUSSION TITLE
Obtainer User/Visitor Time Zone

### DISCUSSION THREAD

**Discussion by mdmontesinos** [oqtane/oqtane.framework#4589] [Q&A]
I was wondering if there's currently any way to obtain the user's browser time zone in Static Render Mode before the page is actually rendered so that I can convert some dates to the local user time and send them back. Perhaps similarly to how the visitor data is gathered.

**Comment by sbwalker** [comment]
@mdmontesinos this is more complicated than you would expect it to be. In order to get the visitor's time zone you need to use JavaScript. I was able to accomplish this with Interactive Blazor (ie. JS Interop) but I have not done it with Static Blazor. If you figure it out, please share with the community.
